static/app/components/replays/replayContext.tsx

```diff
@@ -318,13 +320,7 @@ function ProviderNonMemo({
         return;
       }
 
-      const skipInactive = replayer.config;
-      if (skipInactive) {
```

Comment:
this was causing the bug with ff/no ff prefs not saving properly -- removing this bit seemed to fix it

i also wasn't quite sure what "_when the player resumes playing its not stuck_" meant; scrubbing in ff mode seems to work fine when i removed it🤔

Reply:
I think it might mean that it gets stuck in ff mode even if there is activity.

Reply:
I was playing around with it and it seemed like it might've been stuck in ff mode but I'm not sure, trying to get a video of it now

Reply:
hmm the behavior seems the same on my branch + prod for the replay i'm testing, do you have an example replay link?

Reply:
I tested using this one: https://sentry.sentry.io/replays/554d6125721246de95b56d9715974424. For the prod version, once you hit somewhere with no activity and it ff, if you scrub to a different stop, it will stop ff. However, on your branch if you hit inactivity and it ff, then you scrub to somewhere else, the ff persists. 
On prod:
https://github.com/getsentry/sentry/assets/55311782/eb30068a-9e8f-49e5-8b76-f10f12ee2447
On your branch: 
https://github.com/getsentry/sentry/assets/55311782/649b2adf-31e2-406e-837b-82c9f13bb6db

Reply:
ahh ok i see what you're saying. added this back in -- should be fixed now

Reply:
looks good ty! removing the videos now cuz pii